firestore/scripts/FriendlyEats.Data.js

```diff
@@ -15,71 +15,71 @@
  */
 'use strict';
 
-FriendlyEats.prototype.addRestaurant = function(data) {
-  var collection = firebase.firestore().collection('restaurants');
+FriendlyEats.prototype.addRestaurant = (data) => {
+  const collection = firebase.firestore().collection('restaurants');
   return collection.add(data);
 };
 
-FriendlyEats.prototype.getAllRestaurants = function(render) {
-  var query = firebase.firestore()
+FriendlyEats.prototype.getAllRestaurants = (render) => {
+  const query = firebase.firestore()
     .collection('restaurants')
     .orderBy('avgRating', 'desc')
     .limit(50);
   this.getDocumentsInQuery(query, render);
 };
 
-FriendlyEats.prototype.getDocumentsInQuery = function(query, render) {
-  query.onSnapshot(function (snapshot) {
+FriendlyEats.prototype.getDocumentsInQuery = (query, render) => {
+  query.onSnapshot((snapshot) => {
     if (!snapshot.size) {
       return render();
     }
 
-    snapshot.docChanges().forEach(function(change) {
+    snapshot.docChanges().forEach((change) => {
       if (change.type === 'added' || change.type === 'modified') {
         render(change.doc);
       }
     });
   });
 };
 
-FriendlyEats.prototype.getRestaurant = function(id) {
+FriendlyEats.prototype.getRestaurant = (id) => {
   return firebase.firestore().collection('restaurants').doc(id).get();
 };
 
-FriendlyEats.prototype.getFilteredRestaurants = function(filters, render) {
-  var query = firebase.firestore().collection('restaurants');
+FriendlyEats.prototype.getFilteredRestaurants = (filters, render) => {
+  const query = firebase.firestore().collection('restaurants');
 
   if (filters.category !== 'Any') {
-    query = query.where('category', '==', filters.category);
+    const query = query.where('category', '==', filters.category);
```

Comment:
I think this will break, since we're redeclaring a variable. If you hit this line, it will fail with error `Identifier 'query' has already been declared`

This is a spot where we need to use `let`. Line 50 should say `let query = ...` and then the other lines can just say `query = ...`